test-e2e/NmriumPage.ts

```diff
@@ -36,7 +36,7 @@ export default class NmriumPage {
   }
 
   public async clickTool(id: string) {
-    await this.page.click(`_react=ToolbarItem[id="${id}"]`);
+    await this.page.click(`_react=ToolbarItem[id="${id}"] >> nth=0`);
```

Comment:
You have multiple items with the same id? That feels wrong.

Reply:
It is not the case, but for a reason, I do not fully understand the `_react=` selector retrieves two elements. I realized that this situation occurs when the tooltip is visible. For now, I addressed it this way to fix the other issues, and I plan to return to resolve it. Do you have any suggestions?

![image](https://github.com/cheminfo/nmrium/assets/35760236/add4864a-9c39-470b-8b3c-0be3de84a1e2)

Reply:
Do you have any suggestions to resolve this issue?